manager/src/main/java/com/redhat/service/smartevents/manager/workers/resources/AbstractWorker.java

```diff
@@ -104,6 +130,10 @@ protected T persist(T managedResource) {
         return getDao().getEntityManager().merge(managedResource);
     }
 
+    protected void recordError(Work work, Exception e) {
+        // Do nothing by default
```

Comment:
@manstis Is this is a left-over?

Reply:
No, but I will refactor to an abstract method and override it accordingly in each `Worker` (`ConnectorWorker` does not need to record any errors as it is handled by its parent `ProcessorWorker`).